pootle/static/js/admin/general/components/ContentPreview.css

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
```

Comment:
These are self-contained components, CSS and JS being a separation of technologies, not concerns. Concerns are being kept together.